docs/zkapps/tutorials/11-advanced-account-updates.mdx

```diff
@@ -11,48 +20,38 @@ zkApps can now be deployed to Berkeley Testnet.
 
 :::
 
-:::note
-
-This tutorial was last tested with [SnarkyJS](https://www.npmjs.com/package/snarkyjs) 0.9.6.
-
-:::
-
 # Tutorial 11: Advanced Account Updates
 
-## zkAppManager Accounts and Tokens
-
-## Overview
-
 In the last tutorial, you learned how zkApp transactions are composed of account updates and about the structure of account updates.
 
-In this tutorial, you learn about more advanced features of account updates. Namely tokens, zkApp Manager Accounts, and account update assertions.
+In this tutorial, you learn about more advanced features of account updates: tokens, zkApp manager accounts, and account update assertions.
```

Comment:
A zkApp Manager Account (which is not an official term, its something the original author came up with) controls certain aspects (in this case, it has control over token - send them, distributes them and mints them). 
I dont have any preference re spelling, but I dont mind leaving zkApp Manager in there - its a decent name

Reply:
thanks @Trivo25  is a "zkApp account" the same thing as a "zkApp manager account"? 
see https://docs.minaprotocol.com/zkapps/how-zkapps-work (many mentions of zkApp accounts, esp:
> Each zkApp account provides 8 fields of 32 bytes each of arbitrary storage. 

> When a Mina address contains a verification key, it acts as a zkApp account.

> While the prover function runs in a user's web browser, the verification key lives on-chain for a given zkApp account and is used by the Mina network to verify that a zero knowledge proof has met all constraints defined in the prover. A verification key is required to create a zkApp account.

Reply:
A zkApp account is a smart contract account, whereas a zkApp Manager account is a specific type of smart contract that manages something (eg token)

Reply:
oooh TIL
@Trivo25  check out https://github.com/o1-labs/docs2/pull/674

Reply:
@Trivo25 
Am I correct that the conceptual hierarchy is account -> manager account (and account that manages the characteristics/mechanics of ...something) -> token account (a specific type of manager account that has control over the mechanics of a custom token)

If so, could you give some other examples of what you would consider a manager account?

What I am trying to do here is determine some soft guidelines in answer to the question "when is it appropriate to refer to an account as a manager account?"

The line is a little blurry for me, probably because this is all conceptual